### DISCUSSION TITLE
What are the specific uses of -use_deltas and -crl_download in OpenSSL, and how are they used?

### DISCUSSION THREAD

**Discussion by onepeople158** [openssl/openssl#25167] [Q&A] [answered]
What are the specific uses of -use_deltas and -crl_download in OpenSSL, and how are they used?

**Comment by t8m** [comment, marked as answer]
Citing documentation `If X509_V_FLAG_USE_DELTAS is set delta CRLs (if present) are used to determine certificate status. If not set deltas are ignored.`

`-crl_download` will attempt to download CRLs via the CRL distribution point set in the CA certificate.
